src/main/resources/default/templates/biz/tycho/kb/welcome_de.html.pasta

```diff
@@ -1,19 +1,26 @@
 <div class="pb-4">
-    <img src="/assets/images/login_logo.png" style="max-height: 100px"/>
+    <img src="/assets/images/login_logo.png" alt="@sirius.kernel.info.Product.getProduct().getName()"
+         style="max-height: 100px"/>
```

Comment:
**Convention:** Please put all attributes on separate lines when the line breaks